content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,146 @@ schema: olm.channel
+
+The cue schema for the `olm.composite.catalogs` schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+### Examples
+**catalogs.yaml**
+
+The following example specifies a catalog configuration that defines a catalog named `v1` and the contribution generation logic to only accepting input that uses the [basic template](#basic-template). When using the `opm alpha render-template composite` subcommand the path to this file can be specified with the `-f` option and can be a file path or a URL (the URL must return the raw file contents).
+
+```yaml
+schema: olm.composite.catalogs
+catalogs:
+- name: v1 {{< code_callout 1 >}}
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.24
+    workingDir: catalogs/v1 {{< code_callout 2 >}}
+  builders:
+    - olm.builder.basic {{< code_callout 3 >}}
+```
+
+- {{< code_callout 1 >}} Defines the `v1` catalog configuration
+- {{< code_callout 2 >}} Defines the directory path that all output should be placed for this catalog. In this example, when rendering using the composite catalog all output for the `v1` catalog will be put under the `catalogs/v1/` directory
+- {{< code_callout 3 >}} Defines the allowed builders for the `v1` catalog. In this example the only allowed builder is the basic builder, meaning the only contributions allowed to be rendered for this catalog are basic templates
+
+**contributions.yaml**
+
+The following example specifies a composite template configuration that defines the input, build process, and output of a catalog contribution for the `v1` catalog.
+
+```yaml
+schema: olm.composite
+components:
+- name: v1 {{< code_callout 1 >}}
+  destination:
+    path: my-package {{< code_callout 2 >}}
+  strategy:
+    name: basic
+    template:
+      schema: olm.builder.basic {{< code_callout 3 >}}
+      config:
+        input: components/v1.yaml {{< code_callout 4 >}}
+        output: catalog.yaml {{< code_callout 5 >}}
+```
+
+- {{< code_callout 1 >}} Defines the build process for the `v1` catalog contribution.
+- {{< code_callout 2 >}} Defines the name of the package being contributed and is used to create the package directory
+- {{< code_callout 3 >}} Specifies that the input for the `v1` catalog contribution is a [basic template][#basic-template] so that the contribution can be rendered appropriately
+- {{< code_callout 4 >}} Specifies the input file that should be used to generate the catalog contribution
+- {{< code_callout 5 >}} Defines what the name of the output file should be. This is joined as a path with the  `destination.path` value defined in {{< code_callout 2 >}}. In this example the full output path after rendering would be `my-package/catalog.yaml`
```

Comment:
```suggestion
- {{< code_callout 5 >}} Defines the name of the output file. This is joined as a path with the `destination.path` value defined in {{< code_callout 2 >}}. In this example, the full output path after rendering the template is `my-package/catalog.yaml`
```
Super small nits. The rationale behind avoiding _should_ is that it can imply both an imperative, "You should do this," or a probability, "This should happen." When it reads as a probability, the term can be misconstrued as hedging or a lack of certainty of the outcome.

Reply:
Addressed in https://github.com/operator-framework/olm-docs/pull/286/commits/1cdb364fe5c9f65cbc47ff7effdb3c41bf104cc0

Reply:
/resolve